ui/src/style/tooltip.less

```diff
@@ -36,6 +36,17 @@
     grid-gap: @medium * 0.5;
     align-items: center;
   }
+
+
+  /**
+   * Adding a small amount of horizontal padding to citations so that there's
+   * some separation between the hover state background color and the left and
+   * right edges of the text.
+   */
+  .tooltip-body__citation {
+    padding-left: 4px;
```

Comment:
Style question: Does it make more sense to do units as 'px', 'em', 'rem'?

Reply:
Ah, good question.

The answer is it depends.

If you want the value to adjust as the user adjusts their base font-size (i.e. because they like fonts to be bigger or smaller in general) then you should use `em` or `rem`.

If we want the value to be the same regardless of the user's default font-size, we should use `px`.

I used pixels here to be consistent with the padding set by `@material-ui`, which is using `px` in this case:

![image](https://user-images.githubusercontent.com/801451/71037872-70c06000-20d5-11ea-9203-8b453e74bb7d.png)

Reply:
Works for me, it's fine to me if we keep consistent with @material-ui

Reply:
👍